Utilities/bin_patch.cpp

```diff
@@ -907,6 +907,12 @@ static usz apply_modification(std::vector<u32>& applied, patch_engine::patch_inf
 {
 	const usz old_applied_size = applied.size();
 
+	if (patch.description.find("cellSpurs urgent commands hack") != umax && patch.hash.find("PRX-") != umax)
```

Comment:
please remove this. The software shouldn't have this type of nonsense.

Reply:
It hurts performance and is hack.

Reply:
well then remove it from the wiki man

Reply:
you might as well add all the other old patches that were ever created here.
This is beyond stupid

Reply:
I disagree, virtually all users who played MGS4 had this patch enabled before, the harm has already been done. I will consider removing this code it if the performance benefit won't be considerably large.

Reply:
I think it's bad practise and it has no place here.
It's not our responsibility to care about patches in the first place.

Reply:
What if we disable PRX patching of firmware modules? It doesn't have any other use case AFAIK. PRX patching is used but on game code only, besides this. Also avoids people working around this by renaming the patch.

Reply:
Alternatively, we could delete the patch from the wiki and bump the patch version from 1.2 to 1.3. That would force all users with legacy patch files to update it, removing the unwanted patch.

Reply:
> Alternatively, we could delete the patch from the wiki and bump the patch version from 1.2 to 1.3. That would force all users with legacy patch files to update it, removing the unwanted patch.

Invalidating all patches might be a bit too much. You're inevitably going to get a bunch of people in discord asking why their patches/games aren't working anymore. Disabling firmware patches seems like it would probably be efine.

Reply:
Let's have clear separation of concerns here. It is not the emulator core's duty to police which patches a user is trying to enable. The reliance on patch having an exact string also brings up issues.
The solution here is straightforward - update the wiki to mention the patch is no longer needed and leave it at that.
If the patch works, the user loses some fps in the worst case, not the end of the world.
If the patch doesn't work, the wiki will tell them why and life goes on.